specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/StorageAccountCreateUserAssignedEncryptionIdentityWithCMK.json

```diff
@@ -34,7 +34,11 @@
             "keyname": "wrappingKey",
             "keyversion": ""
           },
-          "keySource": "Microsoft.Keyvault"
+          "keySource": "Microsoft.Keyvault",
+          "identity":{
+            "type": "ManagedServiceIdentity",
+            "userAssignedIdentity": "/subscriptions/{subscription-id}/resourceGroups/res9101/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{managed-identity-name}"
```

Comment:
When se set "userAssignedIdentity" , do we must set `"type": "ManagedServiceIdentity"`
(If we only set "userAssignedIdentity" without  `"type": "ManagedServiceIdentity"`, what will happen?)
If we don't set` "type": "ManagedServiceIdentity"`, is the default `StorageService`? Will server return it in get/list accounts?

Does the "userAssignedIdentity" must in the "identity" list we have set to the account? If we first create the account with keyvault encryption and "userAssignedIdentity"， then update account by remove the used "userAssignedIdentity" from the "identity" list , what will happen?

Reply:
I have replied to these questions in an internal email discussion

Reply:
As it's not clear in swagger, it might take difficulty for SRP user to use the feature.
Could you add the limitation of the parameters to the description of the properties?

Reply:
When we set "userAssignedIdentity" , do we must set "type": "ManagedServiceIdentity"
(If we only set "userAssignedIdentity" without "type": "ManagedServiceIdentity", what will happen?)
Will server return it in get/list accounts?

- No it is defaulted to "ManagedServiceIdentity" when unset. 
- if you only set "userAssignedIdentity", we default it on the server side. 
- Yes. it will return it in GET/LIST.